initial/pom.xml

```diff
@@ -15,6 +15,16 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.springframework.cloud</groupId>
```

Comment:
There is usually little difference. "initial" should represent a bare bones project created from start.spring.io. "complete" is after you add all the code from the guide.

Users should be able to grab "initial", walk through your guide, and arrive at "complete".

Or they should be able to grab "complete" and run everything, looking at what is in the guide to explain it.